packages/driver/src/cy/snapshots.ts

```diff
@@ -147,7 +147,9 @@ export default {
         const isJqueryElement = $dom.isElement($elToHighlight) && $dom.isJquery($elToHighlight)
 
         if (isJqueryElement) {
-          $elToHighlight.attr(HIGHLIGHT_ATTR, true)
+          // JQuery.attr doesn't support `true` as a value.
+          // @ts-ignore
+          ($elToHighlight as JQuery<HTMLElement>).attr(HIGHLIGHT_ATTR, true)
```

Comment:
Since data attribute values are strings, can we update to:
```suggestion
          ($elToHighlight as JQuery<HTMLElement>).attr(HIGHLIGHT_ATTR, 'true')
```

Reply:
wrote it in this way because this breaks the test below:

https://github.com/cypress-io/cypress/blob/ce63723447797e5cebf0f3189057ad49ad09a675/packages/driver/cypress/integration/cy/snapshot_spec.js#L97-L103

After more research, I learned that `data-cypress-el` is an internal attribute and I could be sure that it is not a breaking change. I fixed the code with the test.